[devtools-translate] Portuguese - What's New in DevTools (Chrome 111) (#5388)
GoogleChrome/developer.chrome.com: site/pt/blog/new-in-devtools-111/index.md

title: "O que há de novo no DevTools (Chrome 111)"
authors:
  - jecelynyeen

>   - khaosdoctor

Reviewer: ```suggestion
```

>   - alvarocamillont

Reviewer: ```suggestion
```

<!-- ## Debugging HD color with the Styles pane {: #color } -->

> ## Debugando cores HD no painel de Estilos {: #color }

Reviewer 2: nit: "HD" -> "alta definição"

Author: ```suggestion
## Debugando cores em alta definição no painel de Estilos {: #color }
```

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/3VkOGbbb5qLVvo1A1qSa.png", alt="Resultado do color-mix no painel Computed.", width="800", height="487" %}

<!-- The color picker supports all the new color spaces with more features. For example, click on the color swatch of `color(display-p3 1 0 1)`. A gamut boundary line has also been added, distinguishing between the `sRGB` and `display-p3` gamuts for a clearer understanding of your selected color's gamut. -->
O seletor de cores suporta todos os novos espaços de cores com mais recursos. Por exemplo, clique na amostra de cor de `color(display-p3 1 0 1)`. Uma linha de limite de gama também foi adicionada, distinguindo entre as gamas `sRGB` e `display-p3` para uma compreensão mais clara da gama de cores selecionada.
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/bL6uw8VV4cGuDd9hmAjX.png", alt="A linha de distinção do gama.", width="800", height="657" %}

<!-- DevTools supports converting colors between color formats. Use the **Change Color Format** icon to access the conversion popup, or simply use the `Shift` + click on a color swatch in the **Styles** pane. -->

> O DevTools oferece suporte à conversão de cores entre formatos de cores. Use o ícone **Alterar formato de cor** para acessar o pop-up de conversão ou simplesmente use `Shift` + clique em uma amostra de cor no painel **Estilos**.

Reviewer 2: nit: "O DevTools oferece suporte à conversão de cores entre formatos diferentes.' The current translation aligns with the original text, but it looks slightly better in pt_BR if we avoid the repetition.

Author: ```suggestion
O DevTools oferece suporte à conversão de cores entre formatos diferentes. Use o ícone **Alterar formato de cor** para acessar o pop-up de conversão ou simplesmente use `Shift` + clique em uma amostra de cor no painel **Estilos**.
```

Issues relacionadas: [1073895](https://crbug.com/1073895), [1395782](https://crbug.com/1395782), [1408777](https://crbug.com/1408777),  [1395782](https://crbug.com/1395782),  [1392717](https://crbug.com/1392717), [1382409](https://crbug.com/1382409), [1392054](https://crbug.com/1392054)

<!-- ## Enhanced breakpoint UX {: #breakpoint-redesign } -->
## Experiência de breakpoints redesenhada {: #breakpoint-redesign }

<!-- The redesigned **Breakpoints** pane allows you to have quick access to commonly used features, in particular, deactivating, editing, and removing breakpoints. -->
O painel reprojetado **Breakpoints** permite que você tenha acesso rápido a recursos comumente usados, em particular, desativação, edição e remoção de pontos de interrupção.

<!-- These are some highlights: -->
Essas são algumas novidades:
<!-- - Both pause exception options moved to the **Breakpoints** pane and labeled with text to make it more self-explanatory. -->
- Ambas as opções de exceção de pausa foram movidas para o painel **Breakpoints** e rotuladas com texto para torná-lo mais autoexplicativo.

>     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/KADuAqVdrxxQDg5AYYeA.png", alt="Pause exception options.", width="800", height="518" %}

Reviewer 2: This alt-text is missing a transation. Proposed: "Opções de exceção de pausa."

<!-- - Breakpoints are grouped by file, ordered by line or column numbers, and are collapsible. -->
- Os breakpoints são agrupados por arquivo, ordenados por números de linha ou coluna e são recolhíveis.

>     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/AXJ7IL5aJ6gwbZmnoH39.png", alt="Group breakpoints by file.", width="800", height="454" %}

Reviewer 2: This alt-text is missing a transation. Proposed: "Agrupe breakposts por arquivo."

<!-- - There are new options to deactivate, remove, and edit breakpoints when hovering over a breakpoint or file. -->
- Existem novas opções para desativar, remover e editar breakpoints ao passar o mouse sobre um deles ou arquivo.

>     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/vAQ4YuWGuXYLxXQXt0HG.png", alt="New options to deactivate breakpoints.", width="800", height="496" %}

Reviewer 2: This alt-text is missing a transation. Proposed: "Novas opções para desativar breakpoints."

<!-- - Click the edit breakpoint button to open the breakpoint editor. From here, you can enter the breakpoint condition or switch to a logpoint. -->
- Clique no botão editar breakpoint para abrir o editor. A partir daqui, você pode inserir a condição do breakpoint ou alternar para um ponto de log.

>     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/SrgbxxF8U3s9yzDPV25q.png", alt="Breakpoint edit dialog.", width="800", height="697" %}

Reviewer 3: These have English alt text.

Reviewer 2: Proposal for this one: "Diálogo de edição de breakpoint"

Author: ```suggestion
- Ambas as opções de exceção de pausa foram movidas para o painel **Breakpoints** e rotuladas com texto para torná-lo mais autoexplicativo.
    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/KADuAqVdrxxQDg5AYYeA.png", alt="Opções de breakpoints.", width="800", height="518" %}

<!-- - Breakpoints are grouped by file, ordered by line or column numbers, and are collapsible. -->
- Os breakpoints são agrupados por arquivo, ordenados por números de linha ou coluna e são recolhíveis.
    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/AXJ7IL5aJ6gwbZmnoH39.png", alt="Agrupe breakpoints por arquivo.", width="800", height="454" %}

<!-- - There are new options to deactivate, remove, and edit breakpoints when hovering over a breakpoint or file. -->
- Existem novas opções para desativar, remover e editar breakpoints ao passar o mouse sobre um deles ou arquivo.
    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/vAQ4YuWGuXYLxXQXt0HG.png", alt="Diálogo de edição de breakpoint.", width="800", height="496" %}

<!-- - Click the edit breakpoint button to open the breakpoint editor. From here, you can enter the breakpoint condition or switch to a logpoint. -->
- Clique no botão editar breakpoint para abrir o editor. A partir daqui, você pode inserir a condição do breakpoint ou alternar para um ponto de log.
    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/SrgbxxF8U3s9yzDPV25q.png", alt="Painel de edição do breakpoint.", width="800", height="697" %}
```

<!-- See [JavaScript debugging reference](/docs/devtools/javascript/reference/) to learn how to debug with DevTools. -->
Consulte [Referência de debugging para JavaScript](/docs/devtools/javascript/reference/) para aprender como depurar com o DevTools.

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/48b648b65cd05071d1950e50d0b529ff20294780 #} 
{# https://chromium.googlesource.com/devtools/devtools-frontend/+/b6c8a19b3922ed95818b5751f1b6548724ff868c #}
{# https://chromium.googlesource.com/devtools/devtools-frontend/+/0b955a3533292790168431db4e54906d4b1fa64a #} 
Issues relacionadas: [1407586](https://crbug.com/1407586), [1402891](https://crbug.com/1402891), [1402893](https://crbug.com/1402893)

<!-- ## Customizable Recorder shortcuts {: #recorder } -->
## Atalhos customizáveis no Gravador {: #recorder }

<!-- Use keyboard shortcuts to record and replay user flows quicker. -->
Use atalhos de teclado para gravar e reproduzir os fluxos do usuário mais rapidamente.

<!-- The **Recorder** introduces a few convenient keyboard shortcuts for faster recording and replaying of user flows.  -->

> O **Gravador** apresenta alguns atalhos de teclado convenientes para gravação e reprodução mais rápidas de fluxos de usuários.

Reviewer 2: I see we sometimes reference the DevTools panels or features by the name in english and other times by the translated name. I'd recommend settling on one or the other and keeping consistent across the article.

Author: I think we are using the panel names in all the cases, are we using it inconsistently?

{# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/0a7c428980b8f8a60e83c780e17f6a6f94007493 #} 
{# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/aea51aad7026881ad26ca1b270d12338adb92d79 #} 

Issue relacionada: [1339771](https://crbug.com/bbb)

<!-- ## Better syntax highlight for Angular {: #syntax } -->

> ## Melhor syntax highlight para Angular {: #syntax }

Reviewer 2: nit: "Syntax highlight melhor para Angular"

Author: Hmm, I'm not sure, I think this option makes the text a bit more informal, can you provide a 3rd input @alvarocamillont?

Reviewer 2: I wouldn't block on this discussion (nit is for nitpicking)

<!-- DevTools enhanced the syntax highlighting for Angular HTML templates, making it easier for you to read code and recognize its structure. -->

> O DevTools aprimorou o realce de sintaxe para templates HTML do Angular, facilitando a leitura do código e o reconhecimento de sua estrutura.

Reviewer 2: "realce de sintaxe" / "syntax highlight" - we seem to sometimes use the translated term and others the english one. I'd recommend keeping it consistent.

Author: ```suggestion
O DevTools aprimorou o sintaxe highlighting para templates HTML do Angular, facilitando a leitura do código e o reconhecimento de sua estrutura.
```

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/717953202d0e8463257e09cc3f68a7769fd25300 #}

Issues relacionadas: [1385374](https://crbug.com/1385374),  [1385678](https://crbug.com/1385678)

<!-- ## Reorganize caches in the Application panel {: #cache } -->
## Reorganize caches no painel de Aplicação {: #cache }

<!-- The **Cache Storage** pane can now be found in the **Storage** section of the **Application** panel, while the **Back/forward cache** pane has been moved to the **Background Services** section.  -->
O painel **Cache Storage** agora pode ser encontrado na seção **Armazenamento** do painel **Aplicação**, enquanto o painel **Back/forward cache** foi movido para a seção **Background Services**.

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/q5ZeDvMc3OseF8fQve5A.png", alt="Caches no painel de aplicação.", width="800", height="506" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/adccf1afe5d585b294dee247f5a4982aca8f5f1e  #}

Issue relacionada: [1407166](https://crbug.com/1407166)

<!-- ## Miscellaneous highlights {: #misc } -->
## Outras novidades {: #misc }

<!-- These are some noteworthy fixes in this release: -->

> Mais algumas novidades dignas de nota nessa release:

Reviewer 2: nit: "Mais algumas novidades dignas de atenção nessa versão"

Author: ```suggestion
Mais algumas novidades dignas de atenção nessa versão:
```

- O painel **Elementos** agora foca automaticamente no primeiro elemento correspondente nos resultados da pesquisa. ([1381853](https://crbug.com/1381853))
<!-- - Various fixes to improve the sourcemap and breakpoints reliability. ([508270](https://crbug.com/508270), [1403362](https://crbug.com/1403362), [1403432](https://crbug.com/1403432), [1396298](https://crbug.com/1396298), [1395337](https://crbug.com/1395337), [1405134](https://crbug.com/1405134)) -->
- Várias correções para melhorar a confiabilidade de sourcemaps e dos breakpoints. ([508270](https://crbug.com/508270), [1403362](https://crbug.com/1403362), [1403432](https://crbug.com/1403432), [1396298]( https://crbug.com/1396298), [1395337](https://crbug.com/1395337), [1405134](https://crbug.com/1405134))
<!-- - To better facilitate debugging, DevTools now supports evaluating expressions with private class members. ([1381806](https://crbug.com/1381806)) -->
- Para facilitar melhor o debugging, o DevTools agora oferece suporte à avaliação de expressões com membros de classes privadas. ([1381806](https://crbug.com/1381806))

>      {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/q68fvo870weBVwupujqf.png", alt="Evaluating expressions with private class members.", width="800", height="683" %}

Reviewer 3: the alt here is in English.

Reviewer 2: "Avaliando expressões com membros de classe privados."

Author: ```suggestion
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/q68fvo870weBVwupujqf.png", alt="Avaliando expressões com membros de classe privados.", width="800", height="683" %}
```
